api/ext/moneyformat/currencies.json

```
@@ -325,5 +325,10 @@
         "name": "Tron",
         "divisibility": 6,
         "crypto": true
+    },
+    "GRS": {
+        "name": "GRS",
```

```suggestion
        "name": "Groestlcoin",
```
It should be full name here ideally (I will fix others later)